src/repository.js

```diff
@@ -25,6 +25,10 @@ export class Repository {
   getTransport() {
     if (this.transport === null) {
       this.transport = this.clientConfig.getEndpoint(this.getMeta().fetch('endpoint'));
+
+      if (!this.transport) {
+        throw new Error(`There is no endpoint '${this.getMeta().fetch('endpoint') || 'default'}' is registered.`);
```

Comment:
I was talking about the message :) You're calling it an endpoint, while it's a transport. Also I fixed the sentence for you because you said "is" twice. That's why I gave you the pre-formatted string. Because I'm nice like that. haha